ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/vlan-interfaces.yml

```diff
@@ -94,115 +94,115 @@ vlan_interfaces:
 
-  Vlan42:
+  - id: Vlan42
     description: SVI Description
     shutdown: false
     ip_address_virtual: 10.10.42.1/24
     helpers:
-      10.10.96.151:
+      - ip_address: 10.10.96.151
         source_interface: Loopback0
-      10.10.96.150:
+      - ip_address: 10.10.96.150
         source_interface: Loopback0
-      10.10.64.150:
+      - ip_address: 10.10.64.150
         source_interface: Loopback0
 
 # IPv6 SVI configuration
 
-  Vlan501:
+  - id: Vlan501
     description: SVI Description
     shutdown: false
     ip_address: 10.50.26.29/27
     ipv6_address: 1b11:3a00:22b0:0088::207/127
     ipv6_nd_ra_disabled: true
 
-  Vlan1001:
+  - id: Vlan1001
     description: SVI Description
     shutdown: false
     vrf: Tenant_A
     ip_address_virtual: 10.1.1.1/24
     ipv6_address: a1::1/64
     ipv6_nd_managed_config_flag: true
     ipv6_nd_prefixes:
-      a1::/64:
+      - ipv6_address: a1::/64
         valid_lifetime: infinite
         preferred_lifetime: infinite
         no_autoconfig_flag: true
 
-  Vlan24:
+  - id: Vlan24
     description: SVI Description
     shutdown: false
     ip_address_virtual: 10.10.24.1/24
     ipv6_address: 1b11:3a00:22b0:6::15/64
     ipv6_nd_managed_config_flag: true
     ipv6_nd_prefixes:
-      1b11:3a00:22b0:6::/64:
+      - ipv6_address: 1b11:3a00:22b0:6::/64
         valid_lifetime: infinite
         preferred_lifetime: infinite
         no_autoconfig_flag: true
     ipv6_virtual_router_address: 1b11:3a00:22b0:6::1
 
-  Vlan1002:
+  - id: Vlan1002
     description: SVI Description
     shutdown: false
     vrf: Tenant_A
     ip_address_virtual: 10.1.2.1/24
     ipv6_address: a2::1/64
     ipv6_nd_managed_config_flag: true
     ipv6_nd_prefixes:
-      a2::/64:
+      - ipv6_address: a2::/64
         valid_lifetime: infinite
         preferred_lifetime: infinite
         no_autoconfig_flag: true
     ipv6_nd_ra_disabled: true
 
-  Vlan75:
+  - id: Vlan75
     description: SVI Description
     shutdown: false
     ip_address_virtual: 10.10.75.1/24
     ipv6_address: 1b11:3a00:22b0:1000::15/64
     ipv6_nd_managed_config_flag: true
     ipv6_nd_prefixes:
-      1b11:3a00:22b0:1000::/64:
+      - ipv6_address: 1b11:3a00:22b0:1000::/64
         valid_lifetime: infinite
         preferred_lifetime: infinite
         no_autoconfig_flag: true
     ipv6_virtual_router_address: 1b11:3a00:22b0:1000::1
 
 # IPv6 + helpers + Mcast
 
-  Vlan89:
+  - id: Vlan89
     description: SVI Description
     shutdown: false
     ip_igmp: true
     ip_address_virtual: 10.10.144.3/20
     ipv6_address: 1b11:3a00:22b0:5200::15/64
     ipv6_nd_managed_config_flag: true
     ipv6_nd_prefixes:
-      1b11:3a00:22b0:5200::/64:
+      - ipv6_address: 1b11:3a00:22b0:5200::/64
         valid_lifetime: infinite
         preferred_lifetime: infinite
         no_autoconfig_flag: true
     ipv6_virtual_router_address: 1b11:3a00:22b0:5200::3
     helpers:
-      10.10.96.101:
+      - ip_address: 10.10.96.101
```

Comment:
This knob can also be a hostname. The EOS cli is `ip helper-address` so we could call this `ip_helper_address` or just `ip_helper` as singular of the list name.

Reply:
@carlbuchmann @tgodaA comments?

Reply:
It can be a hostname as well:
<img width="327" alt="image" src="https://user-images.githubusercontent.com/88734745/163575308-bbe39d2f-636f-42ca-8b19-0607cc0a62fd.png">

Reply:
So ```ip_helper``` looks good to me as it can be a server IP or a hostname.

Reply:
I also agree with just `ip_helper`

Reply:
what is to be kept in readme ? --> - ip_helper: < ip_helper_address_1 >

Reply:
yes that is fine. Thanks

Reply:
Updated